docs/source/index.rst

```diff
@@ -64,80 +64,50 @@ For developers
     install/updating_installation
     install/troubleshooting
 
-
-.. toctree::
-    :maxdepth: 1
-    :caption: Getting started
-    :hidden:
-
-    get_started/index
-    get_started/computers
-    get_started/codes
-
 .. toctree::
     :maxdepth: 1
     :caption: Working with AiiDA
     :hidden:
 
-    working_with_aiida/index
-    import_export/index
-
-
-.. toctree::
-    :maxdepth: 1
-    :caption: Terms and Concepts
-    :hidden:
-
-    concepts/provenance
-    concepts/processes
-    concepts/calculations
-    concepts/workflows
+    topics/data
```

Comment:
I feel the order should be something like:

topics/plugins
topics/computers_codes
topics/cli
topics/scripting
topics/data

Caching could be moved into tips&tricks.
Schedulers also doesn't need to be top-level and could be moved there.
Schedulers and Caching could be moved into tips&tricks

Reply:
I have mixed feelings about moving the "Caching" topic to "Tips & Tricks", however, I strongly disagree that "Schedulers" should be part of "Tips & Tricks". That section should be strictly reserved for topics that are not essential to working with AiiDA, however, that does not apply to the topic of "Schedulers" in my opinion.

Finally, I don't think that plugins should be at the very top, but I believe that this would probably need to be hashed out in a separate and larger discussion. For now, I would need clear guidance from senior developers on how to move forward.

@sphuber @giovannipizzi Could you weigh in, please?

Reply:
> Finally, I don't think that plugins should be at the very top

You could move "installing plugins" to the installation section.
it's kind of important, though, to have a plugin since there isn't much you can do with AiiDA without it.

> I strongly disagree that "Schedulers" should be part of "Tips & Tricks". That section should be strictly reserved for topics that are not essential to working with AiiDA

Ideally, users should not have to care about this. The basic configuration of the scheduler is done at the computer setup, i.e. from there one can link to wherever the more detailed explanation is placed.
I agree that "Tips & tricks" is not how I would call this "collection" - but I would also argue that the page on schedulers does not belong at the same level as a section explaining how to use the cli / shell / manage data etc.